### DISCUSSION TITLE
How do i add slash commands that I can use everywhere

### DISCUSSION THREAD

**Discussion by 02840** [discord-jda/JDA#2708] [Questions and Help]
How can i add slash commands that i can use everywhere (user-installable apps)?

**Comment by codenamehaven** [comment]
Hi @02840,👋

Global slash commands are what you need. Register them as application commands by following the official JDA guide here:
- https://jda.wiki/using-jda/interactions/#slash-commands

Think about pinning the guide in your favorites. It's really handy!
